Formula/xorriso.rb

```diff
@@ -13,13 +13,21 @@ class Xorriso < Formula
     sha256 "88442ed4676b021a09bafeef4cd3b40a7e822bb7b4239a1de0518360eefbd5a6" => :mountain_lion
   end
 
+  # Upstream fix for "unknown type name 'ssize_t'"
+  # See http://lists.gnu.org/archive/html/bug-xorriso/2016-05/msg00001.html
+  patch :p0 do
+    url "http://bazaar.launchpad.net/~libburnia-team/libisofs/scdbackup/diff/1319"
+    sha256 "c548c59b143efb87ccb3dc08f80f527ae93918fab56894c9f8876f31dccce142"
+  end
+
   def install
     system "./configure", "--disable-debug", "--disable-dependency-tracking",
                           "--prefix=#{prefix}"
-    system "make", "install"
+    system "make"
+    ENV.deparallelize { system "make", "install" }
```

Comment:
Bizarre and lovely, indeed. I'm amazed at your willingness (and ability) to dissect this. I think it's best to stick with the `ENV.deparallelize` workaround and to forward what you've learned to upstream (if you haven't already), so that they can hopefully fix that for good in future releases.

Gotta love Autotools build systems …